forte/processors/data_augment/algorithms/typo_generator.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import random
+from abc import abstractmethod
+import json
+
+
+__all__ = [
+    "TypoGenerator",
+    "UniformTypoGenerator",
+]
+
```

Comment:
I think we can put the typo generator to the typo_replacement_op folder, so that the files under `algorithms` are all corresponded to one algorithm

Reply:
I think the typo generator and typo_replacement_op are both under algorithm folder. Do you mean I should create a new folder to store this method?